subgraph.template.yaml

```diff
@@ -804,3 +804,111 @@ templates:
         - event: SplitExternalErc20AwardsSet(bool)
           handler: handleSplitExternalErc20AwardsSet
 
+  - name: MultipleWinners_v3_2_0
+    kind: ethereum/contract
+    network: {{network}}
+    source:
+      abi: MultipleWinners
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      file: ./src/mappingForMultipleWinners_v3_1_0.ts
+      entities:
+        - PeriodicPrizeStrategy
+        - MultipleWinners
+      abis:
+        - name: MultipleWinners
+          file: ./abis/v3_2_0/abis/MultipleWinners.json
+        - name: PrizePool_v3
+          file: ./abis/PrizePool_v3.json
+        - name: PrizePool_v3_1_0
+          file: ./abis/v3_1_0/abis/PrizePool.json
+        - name: CompoundPrizePool
+          file: ./abis/v3_1_0/abis/CompoundPrizePool.json
+        - name: ERC20
+          file: ./abis/v3_1_0/abis/ERC20Upgradeable.json
+        - name: ControlledToken
+          file: ./abis/v3_1_0/abis/ControlledToken.json 
+      eventHandlers:
+        - event: TokenListenerUpdated(indexed address)
+          handler: handleTokenListenerUpdated
+        - event: PrizePoolOpened(indexed address,indexed uint256)
+          handler: handlePrizePoolOpened
+        - event: RngServiceUpdated(indexed address)
+          handler: handleRngServiceUpdated
+        - event: OwnershipTransferred(indexed address,indexed address)
+          handler: handleOwnershipTransferred
+        - event: ExternalErc20AwardAdded(indexed address)
+          handler: handleExternalErc20AwardAdded
+        - event: ExternalErc20AwardRemoved(indexed address)
+          handler: handleExternalErc20AwardRemoved
+        - event: ExternalErc721AwardAdded(indexed address,uint256[])
+          handler: handleExternalErc721AwardAdded
+        - event: ExternalErc721AwardRemoved(indexed address)
+          handler: handleExternalErc721AwardRemoved
+        - event: Initialized(uint256,uint256,indexed address,address,address,address,address[])
+          handler: handlePeriodicPrizeInitialized
+        - event: NumberOfWinnersSet(uint256)
+          handler: handleNumberOfWinnersSet
+        - event: PrizePoolAwarded(indexed address,uint256)
+          handler: handlePrizePoolAwarded
+        - event: PrizePoolAwardStarted(indexed address,indexed address,indexed uint32,uint32)
+          handler: handlePrizePoolAwardStarted
+        - event: SplitExternalErc20AwardsSet(bool)
+          handler: handleSplitExternalErc20AwardsSet
+
+
+  - name: MultipleWinners_v3_3_2
```

Comment:
I wonder if we can put this in a yaml or json file (like the `/networks/4.json`) and use mustache to fill it in as variables. Would save a ton of copy and paste and other headaches!